### ISSUE TITLE
Correct `probe.rs/docs/getting-started/installation` package dependencies

### ISSUE THREAD

**Issue by k12ish** [probe-rs/webpage#83]
The probe.rs website reports different dependent packages for Ubuntu compared to the README.md:

```
# probe.rs
sudo apt install -y pkg-config libusb-1.0-0-dev libftdi1-dev
# README.md
sudo apt install -y libusb-1.0-0-dev libftdi1-dev libudev-dev libssl-dev
```

- `libssl-dev` is a required dependency which provides access to openssl
- `lib-udev` is required by the `libudev-sys` crate

This was particularly painful for me because compiling code on my Raspberry Pi takes ~10 minutes, and I ended up recompiling code 3-4 installing missing dependencies.

I couldn't find the source code for `probe.rs`, but I'd be happy writing a PR for it if you could point me to the file.

**Comment by Yatekii** [comment]
Thanks for reporting :) I transferred this issue to the website repository :)

**Comment by Tiwalun** [comment]
Fixed in #85
